.github/workflows/deploy-prod.yaml

```diff
@@ -215,6 +215,41 @@ jobs:
         draft: false
         prerelease: false
 
+    - name: Get npm cache
+      id: npm-cache-dir
+      run: |
+        echo "::set-output name=dir::$(npm config get cache)"
+    - uses: actions/cache@v2
+      id: npm-cache
+      with:
+        path: ${{ steps.npm-cache-dir.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys:
+          ${{ runner.os }}-node-
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+
+    - name: Initialize web
+      if: steps.npm-cache.outputs.cache-hit != 'true'
+      run: |
+        make -C web deps
```

Comment:
does this and below include npm deps in the sbom? i imagine this would not be part of the generated assets included in the release (if we had release assets). what should be included in the sbom?